### PR TITLE
Update add-to-app section with latest information

### PR DESCRIPTION
**Pull request by blasten**
_Description of what this PR is changing or adding, and why:_ The add-to-app section makes references to older versions of Flutter that are no longer relevant at this time.

cc @Hixie

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/development/add-to-app/index.md [hunk 1]
```diff
@@ -16,24 +16,12 @@ app's UI in Flutter. Or, just to run shared Dart logic.
...
-  to the [new Android plugin APIs][Android plugin APIs], based on [`FlutterPlugin`].
-* Plugins that don't support `FlutterPlugin` might have unexpected
-  behaviors if they make assumptions that are untenable in add-to-app
-  (such as assuming that a Flutter `Activity` is always present).
-* As of v1.17, the Flutter module only supports AndroidX applications on Android.
-
-As of Flutter v1.26, add-to-app experimentally supports adding multiple
-instances of Flutter engines, screens, or views into your app. This can
-help integration scenarios such as a hybrid navigation stack with mixed
+add-to-app supports integrating multiple instances of any screen size.
```
Comment (domesticmouse) [inline src/development/add-to-app/index.md:19]: Starting a paragraph with a lower case letter feels odd. Maybe wrap `add-to-app` in backticks?

/cc @sfshaza2

Comment (sfshaza2) [inline src/development/add-to-app/index.md:19]:
```suggestion
The `add-to-app` feature supports integrating multiple instances of any screen size.
```

**Review by sfshaza2 — APPROVED** [review]
lgtm with one minor tweak.

**Comment by blasten** [comment]
Thanks

**Pull request merged**